projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.scss

```diff
@@ -1,34 +1,24 @@
-/* stylelint-disable selector-class-pattern */
-
-// TODO: ACS-8458 Review css selectors
-
-.aca-bulk-actions-select {
-  margin-top: 31px;
-}
+@import '@alfresco/adf-core/lib/styles/mat-selectors';
 
 .aca-bulk-actions-form-field {
   margin-left: 24px;
   margin-top: 1.4375em;
   width: 295px;
 
-  .mat-form-field-flex,
-  .mat-form-field-infix {
+  #{$mat-form-field-flex},
+  #{$mat-form--text-field-infix} {
```

Comment:
Based on this [comment](https://github.com/Alfresco/alfresco-ng2-components/pull/10067/files#r1711065272) you might need to change the name of the variable to `mat-form-text-field-infix`

Reply:
yes, sure